apps/portals/src/configurations/elportal/synapseConfigs/cohortbuilder.ts

```diff
@@ -12,9 +12,62 @@ import {
   handleFilesToParticipants,
   handleSelectedFilesToParticipants,
 } from './handleFilesToParticipants'
+import {
+  QueryWrapperSynapsePlotProps,
+  QueryWrapperSynapsePlotRowClickEvent,
+} from 'synapse-react-client/src/components/QueryWrapperPlotNav/QueryWrapperSynapsePlot'
+import { SynapseUtilityFunctions } from 'synapse-react-client'
 
 const rgbIndex = 1
 
+const handlePlotClick = (event: QueryWrapperSynapsePlotRowClickEvent) => {
+  // window.alert(`Handling click on this row: ${JSON.stringify(event.row)}`)
+  const { setRangeFacetValue } = event.queryContext
+  let newMin: string | undefined = undefined,
+    newMax: string | undefined = undefined
+  const value = event.row.values[0] as string
+  const values = value.match(/\d+/g)
+  if (values && values.length > 0) {
+    newMin = values[0]
+    if (values.length > 1) {
+      newMax = values[1]
+    }
+  }
+  // need to set both for the combined range filter
+  setRangeFacetValue({ columnName: 'minAge' }, newMin, newMax)
+  setRangeFacetValue({ columnName: 'maxAge' }, newMin, newMax)
```

Comment:
I think this filter + the SQL only works in the participant table without extra logic to handle absolute min/max because
1. All of the rows represent a single participant that has a single age
2. The plot bin boundaries match the age bin boundaries in the data.

How is this supposed to work for the files table? You have a file with many participants, where many are under 90 and one is 90-94. I think now that file would be put in the 90-99 bin (since that's the maxAge), and it would be improperly excluded from the all of the other bins. 

I think there's a fundamental issue where we're flattening multiple data points per row to 1 data point and still displaying it like it represents many.

Reply:
You're absolutely right, Nick.  If a file has an age range of 85-95 (for example), it would only be represented in the 90-95 bin.  I think it would also be strange if the file was represented more than once in the plot.
Maybe this plot on the file level is not as important as the one for the participant table.   I'll ask Milan.

Reply:
Confirmed with Milan, this should be on the participants view only.  "Even long term I think it only belongs on the Participant table. We will eventually get and surface animal data in the Files table, and ages won’t make sense across species"